docs/release-notes/8.15.asciidoc

```diff
@@ -1,6 +1,52 @@
 [[release-notes-header-8.15.0]]
 == 8.15
 
+[discrete]
+[[release-notes-8.15.2]]
+=== 8.15.2
+
+[discrete]
+[[known-issue-8.15.2]]
+==== Known issues
+
+// tag::known-issue-189676[]
+[discrete]
+.Tags appear in Elastic AI Assistant's responses
+[%collapsible]
+====
+*Details* +
+On August 1, 2024, it was discovered that Elastic AI Assistant's responses when using Bedrock Sonnet 3.5 may include `<antThinking>` tags, for example `<search_quality_reflection>` ({kibana-issue}189676[#189676]).
+
+====
+// end::known-issue-189676[]
+
+[discrete]
+[[features-8.15.2]]
+==== New features
+* Automatic Import can now use sample input logs to identify log format types ({kibana-pull}190407[#190407]).
+* Allows Automatic Import to use system logs with structured message bodies ({kibana-pull}191749[#191749]).
+
+[discrete]
+[[enhancements-8.15.2]]
+==== Enhancements
+* {elastic-defend} now supports Ubuntu 24.04.
+* {elastic-defend} now supports proxy configuration with {ls} output.
```

Comment:
This change was actually added in 8.15.1, so I'll remove it here and add it to the enhancements section of the 8.15.1 release notes. 

```suggestion
```